src/test/java/it/pagopa/pm/gateway/controller/CreditCardPaymentControllerTest.java

```diff
@@ -67,7 +67,7 @@ public void getPaymentInfoTest() throws Exception {
                 .andExpect(status().isOk());
     }
 
-    private static final String APP_ORIGIN = "APP";
+    private static final String APP_ORIGIN = "ECOMMERCE_WEB";
```

Comment:
Either roll back the change or rename the field, the name and value of a variable must be consistent